diff -Naur tlaps-1.1.1/tlapm/src/backend/prep.ml tlaps-1.1.1-fix/tlapm/src/backend/prep.ml
--- tlaps-1.1.1/tlapm/src/backend/prep.ml	2012-11-22 11:47:58.000000000 +0000
+++ tlaps-1.1.1-fix/tlapm/src/backend/prep.ml	2013-05-03 23:33:36.003597833 +0100
@@ -425,7 +425,7 @@
     ["trivial"],
     prob,
     cleanup prob,
-    "using assms by safe",
+    "using assms by safe\n",
     [Types.Triv]
   end
 ;;
